custom/icds_reports/data_pull/queries.py

```diff
@@ -196,3 +196,14 @@ class HotCookedMealStats(MonthBasedQuery):
 class ChildrenCount(MonthBasedQuery):
     name = "Children Count"
     query_file_path = "custom/icds_reports/data_pull/sql_queries/children_count.sql"
+
+
+class VHSNDMonthlyCount(MonthBasedQuery):
+    name = "VNSND monthly count"
+    query_file_path = "custom/icds_reports/data_pull/sql_queries/vhsnd_monthly_report.sql"
+
+    @cached_property
+    def sql_query(self):
```

Comment:
this method would get removed if you update the sql to use interval instead.

Reply:
done